certora/specs/Basic.spec

```diff
@@ -47,10 +52,12 @@ filtered{f -> !f.isView && f.contract == NodeA}
 
 definition methodsCallEtherNode_NodesManager(method f) returns bool =
     f.selector == sig:EtherFiNodesManager.partialWithdraw(uint256).selector ||
+    f.selector == sig:EtherFiNodesManager.batchPartialWithdraw(uint256[]).selector ||
```

Comment:
I filtered the batch withdraw on purpose because it's simply a multi-call to partial withdraw.
The purpose of the ignored methods was to filter out non-interesting methods.
It's OK that you retained it, but it was supposed to save timeouts.

Reply:
Yep turns out it is not needed to filter this to avoid timeouts, so we may as well keep it to make the rule clearly complete.